### PR TITLE
Fixes #11927: Move common importer attributes to base class

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: lib/runcible/models/yum_importer.rb [hunk 1]
```diff
@@ -4,12 +4,6 @@ class YumImporter < Importer
       ID = 'yum_importer'
       REPO_TYPE = 'rpm-repo'
 
-      #https://github.com/pulp/pulp/blob/master/rpm-support/plugins/importers/yum_importer/importer.py
-      attr_accessor 'feed', 'ssl_verify', 'ssl_ca_cert', 'ssl_client_cert', 'ssl_client_key',
-                    'proxy_url', 'proxy_port', 'proxy_pass', 'proxy_user',
-                    'max_speed', 'verify_size', 'verify_checksum', 'num_threads',
-                    'newest', 'remove_old', 'num_old_packages', 'purge_orphaned', 'skip', 'checksum_type',
-                    'num_retries', 'retry_delay'
```
Comment (parthaa) [inline lib/runcible/models/yum_importer.rb]: ssl_verify is not in base class

Comment (parthaa) [inline lib/runcible/models/yum_importer.rb]: Feels a bit aggressive here . Quite a few fields removed here dont seem to be in the base class

Comment (ehelms) [inline lib/runcible/models/yum_importer.rb]: The fields you mention are ones I couldn't find evidence even exist. For example, ssl_verify is not a thing but ssl_validation is. I tried to dig through all the plugin and base code to find out what I could.

Comment (parthaa) [inline lib/runcible/models/yum_importer.rb]: I think katello sets ssl_verify to true

Comment (ehelms) [inline lib/runcible/models/yum_importer.rb]: My point is, I am fairly certain it got renamed, and a while ago, see:

https://github.com/pulp/pulp_rpm/blob/2ced12bcbd9ea5af0e53ccc96075ba54536eb480/docs/user-guide/release-notes/2.2.x.rst#api-changes

Comment (parthaa) [inline lib/runcible/models/yum_importer.rb]: nice. Hopefully if we have missing attrs here it will get caught ...

Comment (ehelms) [inline lib/runcible/models/yum_importer.rb]: And, grepping katello, I don't see our use of ssl_verify anywhere.

Comment (parthaa) [inline lib/runcible/models/yum_importer.rb]: LGTM then ..

**Comment by parthaa** [comment]
APJ from me if you are able to run katello tests against this runcible PR and they pass

**Pull request merged**